pkg/minikube/mustload/mustload_test.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2020 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mustload
+
+import (
+	"os"
+	"testing"
+)
+
+func TestPartial(t *testing.T) {
+	path := "../config/testdata/profile/.minikube"
+	os.Setenv("MINIKUBE_HOME", path)
```

Comment:
I suggest passing the MINIKUBE_HOME as an optional parameter to the Partial, simmilar to this https://github.com/medyagh/minikube/blob/3d09d8e6ae4dcefcf0d630edaee632d522092734/pkg/minikube/config/profile.go#L108-L109


setting environment variable in testings, will cause race condition since we run many tests in parallel.